zapetw/core_windows.go

```diff
@@ -0,0 +1,39 @@
+package zapetw
+
+import (
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func getETWencoder() zapcore.Encoder {
+	encoderConfig := zap.NewProductionEncoderConfig()
+	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	return zapcore.NewJSONEncoder(encoderConfig)
+}
+
+func getETWCore(eventName string, loggingLevel zapcore.Level) (zapcore.Core, error) {
+	etwSyncer, err := NewETWWriteSyncer(eventName, loggingLevel)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to initialize ETW logger core")
+	}
+	etwcore := zapcore.NewCore(getETWencoder(), zapcore.AddSync(etwSyncer), loggingLevel)
```

Comment:
what if I don't want to use the JSON encoder from `getETWencoder`?

Reply:
Agreed. One of the nice things about structured logging is that the presentation of the structure is entirely incidental. You can swap it out for different formats if you like. We lose this by assuming JSON here.